content/en/install.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Installation
```

Comment:
We could sync this with what we have in the doc. Or link one to the other.

Reply:
Where are the install instructions in the docs?  I found this:
http://scipy.github.io/devdocs/install_upgrade.html

It links to:
https://scipy.org/install.html

The file `content/en/install.md` is basically what was on
https://scipy.org/install.html

Here is what it looks like now:
https://37-407308288-gh.circle-artifacts.com/0/public/install.html

Reply:
This `install.md` needs a rewrite, but let's not do that in this PR. Just minimal changes for outdated stuff, and then merge it. There's a lot to fix up in our install docs, and that should be separate from a new website.

> Where should the official install docs live?

In the SciPy html docs (Sphinx). The page here should be very high-level, even less than what is here now (no Linux distro specific stuff for example).